src/test/java/org/kohsuke/github/GHUserTest.java

```diff
@@ -162,4 +163,10 @@ public void verifyBioAndHireable() throws IOException {
         assertThat(u.getPublicGistCount(), equalTo(4));
         assertThat(u.getPublicRepoCount(), equalTo(96));
     }
+
+    @Test
+    public void verifyLdapDn() throws IOException {
```

Comment:
```suggestion
    public void verifyLdapdn() throws IOException {
```

Reply:
@bitwiseman 
Was this because of the wiremock directory casing? I'm not sure how it passed in my local machine.

Reply:
Thanks for fixing this!

Reply:
Is there any planned release soon?